services/snapshotter/service_test.go

```diff
@@ -163,7 +165,7 @@ func TestSnapshotter_RequestMetastoreBackup(t *testing.T) {
 	}
 	defer l.Close()
 
-	s.MetaClient = &MetaClient{Data: data}
+	s.MetaClient = &MetaClient{data: data.Clone()}
```

Comment:
Are these new `Clone()` calls needed for the tests to pass?

Reply:
Yeah! The `Clone()` calls are unnecessary, which is removed now.